working/Build_RxE.sql

```diff
@@ -122,44 +122,7 @@ JOIN concept c ON c.concept_id = r.concept_id_2
 		'RxNorm',
 		'RxNorm Extension',
 		'UCUM'
-		)
-
-UNION ALL
```

Comment:
Why is it non-needed?

Reply:
Once we made a mistake in one of the RxE releases there's no chance to fix it.
Anyway, when we populate relationship_to_concept we go through existing mappings and synomyms. So this step not only redundant but dangerous.